src/core_plugins/kibana/server/tutorials/apm/get_index_pattern_saved_object.js

```diff
@@ -0,0 +1,42 @@
+    }
+  };
+}
```

Comment:
What about moving this to json, and exporting it like:

```js
const indexPattern = require('./indexPattern.json');
indexPattern.attributes.title = apmIndexPattern;
return indexPattern;
```

(immutable or not)